### PR TITLE
feat: donation banner 2024 integration

### LINKED ISSUE #9264: New donation banner for 2024
### What
A redesigned donation banners (in ENG and FR) on the website for our donation end-of-year campaign

### Why
To make the call-to-donate more obvious. 

### Screenshot/Mockup/Before-After
v1 ENG
V1 FR
v2 ENG
v2 FR

Group photo: 
PXL_20231021_144030096~2

### PR DESCRIPTION
**Pull request by Valimp**
### What

Integration of the new donation banner for 2024

### Screenshot

Capture d’écran 2023-11-28 à 10 27 37

### Related issue(s) and discussion

- Fixes #9264 

@stephanegigandet @raphael0202

### CODE REVIEW

File: po/common/common.pot [hunk 1]
```diff
@@ -5748,8 +5748,16 @@ msgctxt "donation_title"
...
+msgctxt "donation_text_2023_main"
+msgid "Help us make food transparency the norm!"
+msgstr ""
+
+msgctxt "donation_text_2023_secondary"
+msgid "As a non-profit organization, we depend on your donations to continue informing consumers around the world about what they eat."
+msgstr ""
+
+msgctxt "donation_text_2023_tertiary"
+msgid "The food revolution starts with you !"
```
Comment (stephanegigandet) [inline po/common/common.pot:5760]: There's no space before ! in English

```suggestion
msgid "The food revolution starts with you!"
```

File: templates/web/common/includes/donate_banner.tt.html [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+    <p class="donation-banner__main-text">
+        [% lang("donation_text_2023_main") %]
     </p>
-    <p class="donation-banner__content-paragraphs--thank-you">[% lang('thank_you_very_much') %] <span class="material-icons">favorite</span></p>
+    <div class="donation-banner__aside">
+      <div>
+        <p class="donation-banner__secondary-text">[% lang("donation_text_2023_secondary") %]</p>
+        <p class="donation-banner__tertiary-text">[% lang("donation_text_2023_tertiary") %]</p>
+      </div>
+      <a class="donation-banner__donate" href="https://fr.openfoodfacts.org/faire-un-don-a-open-food-facts?utm_source=off&utf_medium=web&utm_campaign=donate-2023-a&utm_term=en-text-button">Donate</a>
```
Comment (stephanegigandet) [inline templates/web/common/includes/donate_banner.tt.html:17]: We should use [% link %] in there I think, we can't hardcode the fr address

**Comment by Valimp** [comment]
I'm going to fix that quickly

**Comment by Valimp** [comment]
there is no rotation on half-logo.svg.
This svg is cut at an angle. I just pick it on the canvas.

**Comment by Valimp** [comment]
@stephanegigandet I think it's all good for the pull request

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Comment by alexgarel** [comment]
Kudos @Valimp :-)

**Pull request merged**